Fullstacker/Api/Controllers/InventoryController.cs

```diff
@@ -0,0 +1,30 @@
+﻿using API.Dtos;
+using DataLayer.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+
+namespace API.Controllers
+{
+    [Route("api/[controller]")]
+    public class InventoryController : ControllerBase
+    {
+        private readonly IInventoryRepository _inventoryRepository;
+
+        public InventoryController(IInventoryRepository inventoryRepository)
+        {
+            _inventoryRepository = inventoryRepository;
+        }
+        /// <summary>
+        /// An endpoint for retrieving a list of items
```

Comment:
"An endpoint for retrieving a list of items"
Doesn't really narrow it down now does it ?